inputs/manual_input.csv

```diff
@@ -317,4 +317,13 @@ offwind-float,investment,2050,1580,EUR/kWel,2020,https://doi.org/10.1016/j.adape
\ No newline at end of file
+offwind-float-connection-underground,investment,2030,1000,EUR/MW/km,2017,Haertel 2017; average + 13% learning reduction,
+organic rankine cycle,lifetime,2020,30,years,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",
+organic rankine cycle,efficiency,2020,0.12,per unit,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551; Breede et al. 2015: Overcoming challenges in the classification of deep geothermal potential, https://eprints.gla.ac.uk/169585/","This is a rough estimate, depends on input temperature, implies ~150 C"
+organic rankine cycle,FOM,2020,2,%/year,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551","Both for flash, binary and ORC plants. See Supplemental Material for details"
+organic rankine cycle,investment,2020,1500,USD/kW_el,2020,Tartiere and Astolfi 2017: A world overview of the organic Rankine cycle market,"Low rollout complicates the estimation, compounded by a dependence both on plant size and temperature."
+geothermal,CO2 intensity,2020,0.12,tCO2/MWh_el,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",Likely to be improved; Average of 85 percent of global egs power plant capacity
+geothermal,lifetime,2020,30,years,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",
+geothermal,efficiency central heat,2020,0.8,per unit,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551; Breede et al. 2015: Overcoming challenges in the classification of deep geothermal potential, https://eprints.gla.ac.uk/169585/","This is a rough estimate, depends on local conditions"
+geothermal,FOM,2020,2,%/year,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",See Supplemental Material of source for details
+geothermal,central heat cost,2020,0.25,%,2020,Frey et al. 2022: Techno-Economic Assessment of Geothermal Resources in the Variscan Basement of the Northern Upper Rhine Graben,"If capital cost of electric generation from EGS is 100%, district heating adds additional 25%"
```

Comment:
```suggestion
geothermal,district heating surcharge,2020,0.25,%,2020,Frey et al. 2022: Techno-Economic Assessment of Geothermal Resources in the Variscan Basement of the Northern Upper Rhine Graben,"If capital cost of electric generation from EGS is 100%, district heating adds additional 25%"
```

Maybe a more comprehensive label? Feel free to disagree.

Reply:
Sounds good, changed the labelling!